shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rule/TableRuleTest.java

```diff
@@ -154,4 +157,29 @@ public void assertActualDataNodesNotConfigured() {
         shardingTableRuleConfig.setTableShardingStrategy(new StandardShardingStrategyConfiguration("shardingColumn", "INLINE"));
         new TableRule(shardingTableRuleConfig, Arrays.asList("ds0", "ds1"), null);
     }
+
+    @Test
+    public void assertDatNodeGroups() {
+        LinkedList<String> dataSourceNames = new LinkedList<>();
+        String logicTableName = "table_0";
+        dataSourceNames.add("ds0");
+        dataSourceNames.add("ds1");
+        TableRule actual = new TableRule(dataSourceNames, logicTableName);
+        Map<String, List<DataNode>> dataNodeGroups = actual.getDataNodeGroups();
```

Comment:
@yangsen00 Maybe we should rename dataNodeGroups to actual. It is the actual result.